replication-api-impl/pom.xml

```diff
@@ -118,6 +118,22 @@
             <groupId>io.micrometer</groupId>
             <artifactId>micrometer-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.jparams</groupId>
+            <artifactId>to-string-verifier</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.awaitility</groupId>
+            <artifactId>awaitility</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.solr</groupId>
```

Comment:
❓ Is this dependency only require for testing? Should it be marked with test scope?

Reply:
It has a scope of test in the dependencyManagement section, does it need it here too?

Reply:
No. I looked but didn't see it. My mistake.